KMM_Showcase/KMM_Showcase/Buttons.swift

```diff
@@ -16,17 +16,20 @@ struct BackButton: View {
         Button(action: {
             self.presentationMode.wrappedValue.dismiss()
         }) {
-            HStack {
+            HStack(spacing: 20) {
                 Image("ic_back")
                     .resizable()
                     .frame(width: 20, height: 20)
                 Text(text)
                     .foregroundColor(.blackColor)
                     .fontWeight(.bold)
+                    .font(.title3)
+                Spacer()
                 
             }
+//            .padding()
             .frame(maxWidth: .infinity)
-            .background(Color.white)
+//            .background(Color.white)
```

Comment:
Can we get rid of the comments for the clean code? 🤔  
Or do you intend to reuse it in the future?

Reply:
forgot to clean it up :sweat_smile: